multitenancy/tenant-manager-client/src/main/java/io/apicurio/multitenant/client/TenantManagerClient.java

```diff
@@ -25,8 +30,11 @@
  */
 public interface TenantManagerClient {
 
+    @Deprecated
     public List<RegistryTenant> listTenants();
 
+    public RegistryTenantList listTenants(TenantStatusValue status, Integer offset, Integer limit, SortOrder order, SortBy orderby);
```

Comment:
Add a javadoc to indicate that the `null` values are expected and valid.

Reply:
done